### PR TITLE
Add improvements to sbt plugin and migrate to `.bloop`

### PR DESCRIPTION
**Pull request by jvican**
I noticed that our `integrationSetUpBloop` was displaying logs with the
paths shortened, when they shouldn't because all the configuration files
are written in the staging sbt directory (because they are source deps).
This commit fixes it.

### CODE REVIEW

File: integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala [hunk 1]
```diff
@@ -60,6 +60,7 @@ object PluginImplementation {
   // We create build setting proxies to global settings so that we get autocompletion (sbt bug)
   val buildSettings: Seq[Def.Setting[_]] = List(
     BloopKeys.bloopInstall := BloopKeys.bloopInstall.in(Global).value,
+    // Bloop users: Do NEVER override this setting as a user if you want it to work
```
Comment (Duhemm) [inline integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala]: This should probably be in the documentation, no?

Comment (jvican) [reply integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala]: We don't have docs for this yet. Do you want me to add them?

Comment (jvican) [reply integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala]: Just added them.

**Comment by jvican** [comment]
I need this for #413 and the JSON configuration changes, so I'm merging for now. @Duhemm Please do raise questions or provide comments if something is not clear. I double checked all the code and the configuration files pass, as well as the community build.

**Pull request merged**